### ISSUE TITLE
How to add drawable to button

### ISSUE THREAD

**Issue by blabz-rahulmandalkar** [cortinico/slidetoact#20]
## What kind of issue is this?

 - [ ] UI Bug. Please provide a Screenshot/Video/GIF of what's the problem. If you need help to record a video you can follow [this guide](https://developer.android.com/studio/command-line/adb.html#screenrecord)

 - [ ] Functional Bug. Please describe the scenario of the bug. Furthermore, please spend the time to write a failing test.

 - [ ] Feature Request. Start by telling what's problem you’re trying to solve. A Pull request is welcome as well.

## Details

In case of bug report, please don't forget to include also:
* Library Version.
* Android Version.
* Emulator/Device specs.
* Logs/Crash reports/Stacktraces.
* Example Code/Link to repositories.

**Comment by cortinico** [comment]
Hey @blabz-rahulmandalkar 
Do you have some mocks or some screenshots of what you want to implement?

The real problem with what you're asking is that the background get resized during the animation. I'm not so sure of what you would love to do with a drawable: crop it? resize it?

Furthermore the `inner_color` would still be needed since it has several effects. From the README.md file:

> Use the inner_color attribute to control the color of the inner circular button, the color of the tick icon and the color of the text. If not set, this attribute defaults to colorBackground from your theme.

**Comment by blabz-rahulmandalkar** [comment]
I want to apply gradient color to inner circular button from drawable resource.

**Comment by squeeish** [comment]
Wrap `SlideToActView` in a `FrameLayout` and set your background drawable to your gradient colour. Add a margin of 1dp to `SlideToActView`, with `outer_color` as #FFFFFF

**Comment by cortinico** [comment]
Yeah that's a bit hacky but it should more or less work. I could eventually implement this feature if someone else is interested in this.

**Comment by cortinico** [comment]
@IreneTrullen I could allow to change the inside icon and maybe the gradient as well. Having a generic drawable for the _bobble_ would be really hard to implement from the animation point of view. Do you have something in mind?

**Comment by IreneTrullen** [comment]
So, maybe can you allow to change the width and height of the icon inside the booble?

**Comment by jamesweb1** [comment]
@cortinico How to set the background of the arrow icon to gradient as the @blabz-rahulmandalkar  mentioned before?

**Comment by cortinico** [comment]
> @cortinico How to set the background of the arrow icon to gradient as the @blabz-rahulmandalkar mentioned before?

Currently is not possible. I'm happy to review a PR if you're willing to develop this feature 🙏

**Comment by samk17cmutpm** [comment]
Hi all, how about this?? Any update?? I need the innerColor could be drawble
<img width="134" alt="Screen Shot 2020-04-06 at 5 21 54 PM" src="https://user-images.githubusercontent.com/7029528/78548534-25b88d00-782b-11ea-8ba3-bded4b3e534d.png">

**Comment by cortinico** [comment]
> I'm happy to review a PR if you're willing to develop this feature 🙏
